onnx/defs/math/old.cc

```diff
@@ -1294,5 +1295,116 @@ ONNX_OPERATOR_SET_SCHEMA(
           result_shape.mutable_dim(static_cast<int>(axis))->set_dim_value(k);
           updateOutputShape(ctx, 0, result_shape);
           updateOutputShape(ctx, 1, result_shape);
-        }));		
+        }));
+
+static const char* TopK_ver10_doc = R"DOC(
+Retrieve the top-K elements along a specified axis. Given an input tensor of
```

Comment:
Is the sorting algorithm stable?

Reply:
yes it is stable sort, i.e. to use axis indices as a tiebreaker for equal values.